superset/commands/exceptions.py

```diff
@@ -44,7 +44,7 @@ def __init__(
         super().__init__(
             _(
                 self.message_format.format(
-                    object_type, '"%s" ' % object_id if object_id else ""
+                    object_type, f'"{object_id}" ' if object_id else ""
```

Comment:
Just wondering whether the trailing space should be there. I realize it was previously, but I'm not sure if that was a typo.